pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.6.2</version>
+        <version>2.6.6</version>
```

Comment:
Please review possibility of usage 2.6.7 version of spring-boot-starter-parent. cc @julianladisch

Reply:
Yes, made the version to 2.6.7 as folio-spring-base.version 3.0.3 has the version 2.6.7 changes were made as part of FOLSPRINGB-51